packages/console/src/pages/Users/components/CreateForm/index.tsx

```diff
@@ -98,23 +123,45 @@ function CreateForm({ onClose, onCreate }: Props) {
         onClose={onClose}
       >
         <form>
-          <FormField isRequired title="users.create_form_username">
+          <FormField title="user_details.field_name">
+            <TextInput {...register('name')} error={errors.name?.message} />
+          </FormField>
+          <FormField title="user_details.field_email">
+            <TextInput
+              {...register('primaryEmail', {
+                validate: (value) =>
+                  validateIdentifier(value, emailRegEx, t('errors.email_pattern_error')),
+                onChange: () => {
+                  revalidateForm();
+                },
+              })}
+              error={errors.primaryEmail?.message}
+            />
+          </FormField>
+          <FormField title="user_details.field_phone">
+            <TextInput
+              {...register('primaryPhone', {
+                validate: (value) =>
+                  validateIdentifier(value, phoneRegEx, t('errors.phone_pattern_error')),
+                onChange: () => {
+                  revalidateForm();
+                },
+              })}
+              error={errors.primaryPhone?.message}
+            />
+          </FormField>
```

Comment:
do we have desc or should we use phone number component (with region number) to tell users that phone numbers should always come with the leading region number?
IMO phone number's region number is mandatory in Logto core.

Reply:
Good call. Update the `phoneRegEx` to `phoneInputRegEx` temporarily, and we will add a phone number input to ensure the user will set up a phone number correctly. tracked by LOG-5942

Reply:
maybe we can use libphonenumber-js to validate the phone number since this can be more strict? You may find some sample code snippets in @logto/console.

Reply:
Using `phoneInputRegEx` is the existing solution in the Console (used in SMS Connector testing), and it's sufficient to meet our needs for now.
We will have a `PhoneNumberInput` component in the near future, but until then, I don't think it's necessary to build another temporary solution by using `libphonenumber-js`.